core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -133,6 +145,50 @@ public Response deleteBucket(@PathParam("bucket") final String bucket) {
   }
 
   /**
+   * @summary gets a bucket and lists all the objects in it
+   * @param bucket the bucket name
+   * @param continuationToken the optional continuation token param
+   * @param maxKeys the optional max keys param
+   * @param prefix the optional prefix param
+   * @return the response object
+   */
+  @GET
+  @Path(BUCKET_PARAM)
+  @ReturnType("ListBucketResult")
+  // TODO(chaomin): consider supporting more request params like prefix and delimiter.
+  public Response getBucket(@PathParam("bucket") final String bucket,
+      @QueryParam("continuation-token") final String continuationToken,
```

Comment:
nit: put these query params and path params into constants because other methods may also need them

Reply:
so far only `bucket` is shared with other methods. I think we should follow the convention in `FileSystemMasterClientRestServiceHandler`.